smac/utils/validate.py

```diff
@@ -237,31 +244,35 @@ def validate_epm(self, config_mode:Union[str, typing.List[Configuration]]='def',
             number of repetitions in nondeterministic algorithms
         runhistory: RunHistory
             optional, RunHistory-object to reuse runs
+        reuse_epm: bool
+            if true (and if self.epm), reuse epm to validate runs
```

Comment:
It would be great if you could add some backticks here to format it python-like.